### PR TITLE
Moving machine set ifevals into snippets

### PR DESCRIPTION
**Pull request by jeana-redhat**
Removed install proof of concept, so this will not actually render any preview

### CODE REVIEW

**Review by bscott-rh — APPROVED** [review]
LGTM with a couple tiny nits and a question.

File: snippets/machine-set-ifeval-close.adoc [hunk 1]
```diff
@@ -0,0 +1,208 @@
+// Text snippet included in the following assemblies:
+//
+// *
```
[the remark this answers is not in the document]
Comment (jeana-redhat) [reply snippets/machine-set-ifeval-close.adoc:3]: Good point - I grabbed it from an existing one but "modules" makes more sense here

File: snippets/machine-set-ifeval-close.adoc [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+//
+// Closes the ifeval directives that are opened by snippets/machine-set-ifeval-open.adoc
+// You must include both snippets in any module that uses them.
+//
+// Use: 
+//
+//  -----------------------------------------------------
+//  ¦ // Module included in the following assemblies:   ¦
+//  ¦ //                                                ¦
+//  ¦ // * path/to/assembly.adoc                        ¦
```
[the remark this answers is not in the document]
Comment (jeana-redhat) [reply snippets/machine-set-ifeval-close.adoc:17]: this one is a sample module so this is correct as-is (module would be included in an assembly)

File: snippets/machine-set-ifeval-open.adoc [hunk 3]
```diff
@@ -0,0 +1,452 @@
...
+// # Control plane machine set opening ifeval directives #
+// #######################################################
+
+// Agnostic
+// Adding a section to evaluate whether this would be helpful for the base specs.
+
+ifeval::["{context}" == "cpmso-config-options-CLOUD"]
+:cpmso:
+endif::[]
+ifdef::cpmso[]
```
[the remark this answers is not in the document]
Comment (jeana-redhat) [reply snippets/machine-set-ifeval-open.adoc:40]: Good call!

**Pull request merged**